inbm/integration-reloaded/scripts/create_signature.py

```diff
@@ -18,12 +18,12 @@
     if num_params == 4:
         password = sys.argv[3].encode('utf-8')
 
-with open(package_name) as package:
-    checksum = hashlib.sha384(package.read().encode('utf-8')).hexdigest()
+with open(package_name, 'rb') as package:
```

Comment:
Ensure that the script handles the case where the file to be signed does not exist or is not accessible. [medium]

Reply:
this is a utility script used in integration reloaded--throwing an exception is fine